packages/core/src/navigation/navigation-group.element.ts

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { html, internalProperty, LitElement, PropertyValues } from 'lit-element';
+
+import {
+  Animatable,
+  animate,
+  AnimationNavigationGroupOpenName,
+  baseStyles,
+  createId,
+  event,
+  EventEmitter,
+  i18n,
+  I18nService,
+  id,
+  property,
+  querySlot,
+  querySlotAll,
+  reverseAnimation,
+  syncProps,
+} from '@cds/core/internal';
+import { styles } from './navigation-group.element.css.js';
+import { NavigationLayout } from './interfaces/navigation-layout.interface.js';
+import { DEFAULT_NAVIGATION_LAYOUT } from './utils/index.js';
+import { CdsNavigationItem } from './navigation-item.element';
+import { CdsNavigationStart } from './navigation-start.element';
+
+export const CdsNavigationGroupTagName = 'cds-navigation-group';
+
+/**
+ *
+ * ```typescript
+ * import '@cds/core/navigation/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-navigation-group>
+ *
+ * </cds-navigation-group>
+ * ```
+ *
+ * @element cds-navigation-group
+ * @slot
+ */
```

Comment:
We should add the props and events here so we can expose them in Storybook

Reply:
I updated all of the element annotations with the cssprop changes I made and added missing events.